samples/csharp-language/expression-trees/ExpressionTreeBuildingSampleOne.cs

```diff
@@ -27,9 +28,8 @@ public override void Run()
             var ySquared = Expression.Multiply(yParameter, yParameter);
             var sum = Expression.Add(xSquared, ySquared);
 
-            Expression<Func<double, double>> sqrt = (x) => Math.Sqrt(x);
-            var methodCall = sqrt.Body as MethodCallExpression;
-            var distance = methodCall.Update(default(Expression), new List<Expression> { sum });
+            var sqrtMethod = typeof(Math).GetMethod("Sqrt", new[] { typeof(double) });
+            var distance = Expression.Call(sqrtMethod, sum);
```

Comment:
It is supported. Except it's [an extension method in the `System.Reflection` namespace](https://dotnet.github.io/api/System.Reflection.TypeExtensions.html#System_Reflection_TypeExtensions_GetMethod_System_Type_System_String_System_Type___), which is why I also added the `using`.